adobe-apiplatform/user-sync.py
# Update umapi.py - fixes #793

fixes #793

## Summary
* makes possible creation of T3 accounts in a single createFderatedID call, without a chained update call

## Testing Steps
* make sure you create a build by incorporating also the change from the enhancement 102 on the umapi-client.py 
* use a source of directory for which username != email and both are email formatted values
* run the tool in this setup and observe the createFederatedID no longer mention 'update' action and also the accounts get created with email != username in admin console  


Fixes #793
